services/QuillLMS/app/models/user.rb

```diff
@@ -868,6 +868,10 @@ def unlink_clever_and_google_accounts!
     update!(clever_id: nil, google_id: nil, signed_up_with_google: false)
   end
 
+  def unlink_google_account!
+    update!(google_id: nil, signed_up_with_google: false)
```

Comment:
Out of scope for this, but I think at some point, we should clean up / remove the "signed_up_with_google" attribute and just rely on `google_id`. Seeing code like this really shows that it's redundant (I agree with setting it to `false` here because there's a lot of logic tied to it, but if we are not keeping it as `true` here, I'm not sure there's a scenario in which it stores different info than `google_id`.

Reply:
That's a good point.  AFAICT it's a redundant attribute that was used for something in the past. 

I will add it to my growing list of things to clean up at some point.